lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -29,7 +29,9 @@
                                 model: userAccountModel,
                                 title: gettext('Username'),
                                 valueAttribute: 'username',
-                                helpMessage: gettext('The name that identifies you on the edX site. You cannot change your username.')
+                                helpMessage: interpolate_text(
+                                    gettext('The name that identifies you on the {platform_name} site. You cannot change your username.'), {platform_name: platformName}
```

Comment:
There might - especially if the microsite's name starts with "the". Maybe this?

The name that identifies you throughout {platform_name}. You cannot change your username.